libraries/stdlib/samples/test/samples/text/strings.kt

```diff
@@ -79,4 +79,196 @@ class Strings {
         assertPrints(byteArray.toString(charset), "Hello")
     }
 
+
+    //Indices is IntRange
+    @Sample
+    fun slice() {
+        assertPrints("abcdef".slice(IntRange(1,2)),"bc")
+        assertPrints("abcdef".slice(IntRange(0,0)),"a")
```

Comment:
Usually `IntRange` is constructed with the `..` operator: `1..2`

Reply:
Also in `1..2` range, the end index 2 can be confused with the range length which also happens to be 2.